frontend/model/notifications/templates.js

```diff
@@ -24,14 +25,22 @@ export default ({
     if ([GIMessage.OP_ACTION_ENCRYPTED, GIMessage.OP_ACTION_UNENCRYPTED].includes(opType) && value) {
       action = value.action
     }
+    const state = sbp('state/vuex/state')
+    let who
+
+    if (message.innerSigningKeyId()) {
+      const innerSigningContractID = findContractIDByForeignKeyId(state[message.contractID()], message.innerSigningKeyId())
+      who = innerSigningContractID && `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${innerSigningContractID}`
+    }
 
     return {
-      body: L("{errName} during {activity} for '{action}' to '{contract}': '{errMsg}'", {
+      body: L(who ? "{errName} during {activity} for '{action}' from {b_}{who}{_b} to '{contract}': '{errMsg}'" : "{errName} during {activity} for '{action}' to '{contract}': '{errMsg}'", {
```

Comment:
The `L` function is not allowed to have a ternary operator inside of it. It is only allowed to have a **string literal** and (optionally) an object for the arguments inside of the string literal.